Dockerfile

```diff
@@ -12,19 +12,22 @@ ENV SCREEN_HEIGHT 1080
 ENV SCREEN_WIDTH 1920
 
 # Set number of threads for parallel execution
-# By default, no parallelisation
+# By default, no parallelization
```

Comment:
This was not a mistake, it is British English, please revert.

Reply:
done